CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 ### 📦 Dependency Updates
 * [AUTOMATIC BUMP] Updates purchases-hybrid-common to 13.17.1 (#628) via RevenueCat Git Bot (@RCGitBot)
 
+## 6.2.2
+## RevenueCat SDK
+### 📦 Dependency Updates
+* [AUTOMATIC BUMP] Updates purchases-hybrid-common to 13.16.2 (#625) via RevenueCat Git Bot (@RCGitBot)
+* [AUTOMATIC BUMP] Updates purchases-hybrid-common to 13.16.0 (#624) via RevenueCat Git Bot (@RCGitBot)
```

Comment:
I believe the second line doesn't add any useful info? But since all this is automated, it's ok I guess

Reply:
Yeah agreed... Since I'm adding this manually, I might as well remove it 👍